docs/howtos/convert.rst

```diff
@@ -8,4 +8,43 @@ One way to do this is from the command line:
 
     $ pyro convert CGYRO "my_gs2_file.in" -o "input.cgyro"
 
-Pyrokinetics will automatically detect the type of the input file.
+Pyrokinetics will automatically detect the type of the input file. Note
+that when converting between codes only the data stored in `LocalGeometry`,
+`LocalSpecies` and `Numerics` is transferred over. Any other CGYRO input
+parameters are taking from a template file which is taking from `pyrokinetics.template_dir`
```

Comment:
```suggestion
parameters are taking from a template file in `pyrokinetics.template_dir`
```

Worth mentioning how to change this template?

Reply:
I do mention that later on, is that what you meant?

Reply:
Oh yes, ignore me, sorry!